src/containers/app/AppContainer.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { MemoryRouter } from 'react-router-dom';
+import { AppContainer, HelloWorldComponent } from './AppContainer';
```

Comment:
Why are you not using the default export? Does it have something to do with it being connected to redux?

Reply:
The default export expects to connect with a redux store. Instead of creating a mock redux store and anything else required, I've elected to test the raw, unconnected component instead.